crates/oxc_prettier/Cargo.toml

```diff
@@ -20,6 +20,8 @@ oxc_allocator = { workspace = true }
 oxc_ast       = { workspace = true }
 oxc_syntax    = { workspace = true }
 oxc_span      = { workspace = true }
+serde         = { workspace = true }
+serde_json    = { workspace = true }
```

Comment:
Not required by this crate

Reply:
https://github.com/oxc-project/oxc/pull/1350/commits/0e8ff9da0d36d59a8531da7e62bb425743074ded